Templates/bootstrap4_template.html

```diff
@@ -0,0 +1,218 @@
+            text-align: center;
+        }
+    </style>
+
+</head>
+<!--
+  The four types of pages used in conditional properties are:
+    home     = the single master home page
+               rendered as "$TargetDir/index.html"
+    contents = the single master index/contents page
+               rendered as "$TargetDir/contents.html"
+    module   = the module index pages, one per module
+               rendered as "$TargetDir/$nsName/$modName/index.html"
+    function = the function pages, one per exported function
+               rendered as "$TargetDir/$nsName/$modName/$funName.html"
+  -->
+
+<body>
+    <nav class="navbar navbar-toggleable-md navbar-inverse bg-inverse fixed-top">
+        <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+            <ul class="navbar-nav mr-auto">
+                {ifdef home}
+                <li class="nav-item active">
+                    <a class="nav-link" href="index.html">API Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="contents.html">API Index</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link disabled" href="#">Module</a>
+                </li>
+                {endif home}
+                
+                {ifdef contents}
+                <li class="nav-item">
+                    <a class="nav-link" href="index.html">API Home</a>
+                </li>
+                <li class="nav-item active">
+                    <a class="nav-link" href="contents.html">API Index <span class="sr-only">(current)</span></a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link disabled" href="#">Module</a>
+                </li>
+                {endif contents}
+                
+                {ifdef module}
+                <li class="nav-item">
+                    <a class="nav-link" href="../../index.html">API Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="../../contents.html">API Index</a>
+                </li>
+
+                <li class="nav-item active">
+                    <a class="nav-link" href="index.html">Module <span class="sr-only">(current)</span></a>
+                </li>
+                {endif module}
+                
+                {ifdef function}
+                <li class="nav-item">
+                    <a class="nav-link" href="../../index.html">API Home</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="../../contents.html">API Index</a>
+                </li>
+
+                <li class="nav-item">
+                    <a class="nav-link" href="index.html">Module</a>
+                </li>
+                {endif function}
+            </ul>
+            <div class="form-inline my-2 my-lg-0">
```

Comment:
The `my-...` notation is a like a mini-expression [language](https://v4-alpha.getbootstrap.com/utilities/spacing/) in Bootstrap.  While I would be happy to do so, I think it would take a bit to explain it here, so I'm not sure it's the right place to do so.  I'm thinking opening comments and docs/example references at the top of the template should be a good place for folks to start if they want to dig in further?